config/locales/views/admin/en.transaction_search.yml

```diff
@@ -9,6 +9,7 @@ en:
       actions:
         reassign_chart_strings: Reassign Chart Strings
         send_notifications: Send Notifications
+        send_zero_dollar_notifications: Send $0 emails?
```

Comment:
Can we double check with the client and make sure they're good with this wording? I definitely prefer this positive phrasing rather than the negative "suppress", but let's make sure NU is on board with it.

Reply:
Good call, I'll check.